### PR TITLE
Update AboutUs with a simple template

### PR DESCRIPTION
**Pull request by JeffZincatz**
Update AboutUs with a simple template
- Add me as a member
- Add my photo
- Add my portfolio

### CODE REVIEW

**Comment by seaweediman** [comment]
Can you fix the failed checks?

File: docs/AboutUs.md [hunk 1]
```diff
@@ -29,31 +19,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-* Responsibilities: Dev Ops + Threading
+<img src="docs/images/jiheng.png" width="200px">
 
-### James Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/JeffZincatz)] [[portfolio](team/johndoe.md)]
```
Comment (seaweediman) [inline docs/AboutUs.md:26]: For portfolio, shouldn't it be team/jiheng.md

Comment (JeffZincatz) [reply docs/AboutUs.md:26]: Oh you are right. Changing it now.

**Review by seaweediman — APPROVED** [review]
LGTM

**Pull request merged**